tests/integration/conftest.py

```diff
@@ -160,13 +161,23 @@ async def app_fixture(
             config=app_config,
             series="focal",
         )
+
+    await model.wait_for_idle()
+
+    # configure postgres
+    await postgres_app.set_config(
+        {
+            "plugin_hstore_enable": "true",
+            "plugin_pg_trgm_enable": "true",
+        }
+    )
     await model.wait_for_idle()
 
     # Add required relations
     unit = model.applications[app_name].units[0]
     assert unit.workload_status == WaitingStatus.name  # type: ignore
     await asyncio.gather(
-        model.add_relation(app_name, "postgresql-k8s:db-admin"),
+        model.add_relation(app_name, "postgresql-k8s:database"),
```

Comment:
I'd suggest you switch to the latest/edge redis, as the revision in stable has some stability issues that might cause flakiness

Reply:
Switched to latest/edge